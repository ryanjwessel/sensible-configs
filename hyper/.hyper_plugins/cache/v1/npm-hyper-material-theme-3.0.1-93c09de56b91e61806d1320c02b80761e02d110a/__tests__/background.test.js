/* globals describe, it, expect */

const {backgrounds} = require('../backgrounds');
const {themes} = require('../default.config');

describe('Backgrounds', () => {
  it('should get the correct background (no opacity)', () => {
    for (const theme in themes) {
      if (theme) {
        expect(backgrounds()[theme]).toMatchSnapshot();
      }
    }
  });

  it('should get the correct background (with opacity)', () => {
    for (const theme in themes) {
      if (theme) {
        expect(backgrounds(0.8)[theme]).toMatchSnapshot();
      }
    }
  });
});
