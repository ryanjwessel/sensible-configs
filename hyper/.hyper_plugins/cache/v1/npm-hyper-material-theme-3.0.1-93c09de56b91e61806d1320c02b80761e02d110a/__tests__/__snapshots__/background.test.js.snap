// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Backgrounds should get the correct background (no opacity) 1`] = `"rgba(41, 45, 62, 1)"`;

exports[`Backgrounds should get the correct background (no opacity) 2`] = `"rgba(33, 33, 33, 1)"`;

exports[`Backgrounds should get the correct background (no opacity) 3`] = `"rgba(9, 11, 16, 1)"`;

exports[`Backgrounds should get the correct background (no opacity) 4`] = `"rgba(38, 50, 56, 1)"`;

exports[`Backgrounds should get the correct background (with opacity) 1`] = `"rgba(41, 45, 62, 0.8)"`;

exports[`Backgrounds should get the correct background (with opacity) 2`] = `"rgba(33, 33, 33, 0.8)"`;

exports[`Backgrounds should get the correct background (with opacity) 3`] = `"rgba(9, 11, 16, 0.8)"`;

exports[`Backgrounds should get the correct background (with opacity) 4`] = `"rgba(38, 50, 56, 0.8)"`;
